scripts/release/release.sh

```diff
@@ -109,6 +109,31 @@ if [ $? != 0 ]; then
   exit 1
 fi
 
+# set up python environment for releasing to pypi
+pushd adam-python
+rm -rf release-venv
+virtualenv release-venv
+. release-venv/bin/activate
+pip install pyspark
```

Comment:
I don't have pip installed by default, does that come in the virtualenv?

Reply:
Correct; virtualenvs include their own pip.